R/epi_workflow.R

```diff
@@ -326,3 +330,37 @@ print.epi_workflow <- function(x, ...) {
   print_postprocessor(x)
   invisible(x)
 }
+
+
+#' Produce a forecast from an epi workflow
+#'
+#' @param epi_workflow An epi workflow
+#' @param fill_locf Logical. Should we use locf to fill in missing data?
+#' @param n_recent Integer or NULL. If filling missing data with locf = TRUE,
+#' how far back are we willing to tolerate missing data? Larger values allow
+#' more filling. The default NULL will determine this from the the recipe. For
+#' example, suppose n_recent = 3, then if the 3 most recent observations in any
+#' geo_value are all NA’s, we won’t be able to fill anything, and an error
+#' message will be thrown. (See details.)
+#' @param forecast_date By default, this is set to the maximum time_value in x.
+#' But if there is data latency such that recent NA's should be filled, this may
+#' be after the last available time_value.
+#'
+#' @return A forecast tibble.
+#'
+#' @export
+forecast <- function(epi_workflow, fill_locf = FALSE, n_recent = NULL, forecast_date = NULL) {
+  if (!epi_workflow$trained) {
+    cli_abort("The epi_workflow is not trained.")
```

Comment:
Best to use the [cli formatting](https://cli.r-lib.org/articles/semantic-cli.html#inline-lists-of-items)